DigitalLearningSolutions.Data/Models/CustomPrompts/CoursePrompt.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class CoursePrompt
+    {
+        public CoursePrompt(int coursePromptNumber, string text, string? options, bool mandatory)
+        {
+            CoursePromptNumber = coursePromptNumber;
+            CustomPromptText = text;
+            Options = SplitOptionsString(options);
+            Mandatory = mandatory;
+        }
+
+        public int CoursePromptNumber { get; set; }
+        public string CustomPromptText { get; set; }
+        public List<string> Options { get; set; }
+        public bool Mandatory { get; set; }

```

Comment:
Mandatory isn't really a thing on course prompts. Seeing this here, I think the CustomPrompt class this has been split from is really a CentreRegistrationPrompt (with our shiny new naming system), so this class, in turn, is probably actually a CourseAdminField. Though I haven't looked at how it's used. Think that's worth bearing in mind when adopting the new names.

Reply:
Removed Mandatory from CoursePrompt and renamed it to CourseAdminField. Renamed CustomPrompt to CentreRegistrationPrompt. CoursePrompt and CustomPrompt now only refer to the prompts themselves in the db, not associated with a centre or course.